packages/syft/src/syft/types/dicttuple.py

```diff
@@ -236,7 +236,16 @@ def __getitem__(self, __key: _KT | slice | SupportsIndex, /) -> _VT | Self:
         return super().__getitem__(self.__mapping[__key])
 
     def __repr__(self) -> str:
-        return f"{self.__class__.__qualname__}{super().__repr__()}"
+        # return f"{self.__class__.__qualname__}{super().__repr__()}"
+        return "[]"
+
+    def _repr_html_(self) -> str:
+        super_repr_html = getattr(super(), "_repr_html_", None)
+        if super_repr_html is None:
+            return super().__repr__()
+        if super_repr_html() == self.__repr__():  # empty DictTuple
+            return "[]"
```

Comment:
Checking for empty DictTuple with `len(self) == 0` is better. So something like
```py
if len(self) == 0 or super_repr_html is None:
    return "0 records"
```

Reply:
Thank you @kiendang very much for the review. Fixed accordingly